ci-operator/README.md

```diff
@@ -0,0 +1,154 @@
+ci-operator
+===========
+
+
+End-to-end tests
+----------------
+
+The preferred way to write e2e tests is using `ci-operator`.  See the
+documentation for details on how to download, build, and execute it:
+
+https://github.com/openshift/ci-operator.git
+
+The first step is writing a configuration file for `ci-operator` for the
+repository.  Many projects already have one, which takes care of most of the CI
+process: downloading the source, building binaries, building RPMs, creating
+images, executing unit tests, etc.  These can be built upon for e2e test,
+requiring little or no modification.
+
+After that, to add an e2e test:
+
+1. Determine the pre-requisites for the test.  Practically, this means choosing
+   the template that is going to be used by `ci-operator`.  This directory
+   already contains templates for the common cases, see
+   [Using a template](#using-a-template) below.
+2. Determine and configure the template's inputs.  This is specific to each
+   template and should be documented in its parameters.  The e2e test might
+   need minor modifications to fit the environment created by the template.
+3. Add one or more prow jobs to prow's configuration file with the information
```

Comment:
Nit: Proper names start with capital letters: prow --> Prow. This seems minor, but from experience I know it helps many people who aren't super-fluent in English.

Reply:
Also here to differentiate between Prow jobs (config stanzas) with `ProwJob`s the CRD object...